src/inscribeit/contract_builder.hpp

```diff
@@ -55,6 +55,9 @@ class ContractBuilder
     ContractBuilder& operator=(const ContractBuilder& ) = default;
     ContractBuilder& operator=(ContractBuilder&& ) noexcept = default;
 
+    virtual std::vector<CMutableTransaction> getTransactions() = 0;
+    virtual CAmount getWholeFee(CAmount fee_rate);
```

Comment:
I suggest removing the fee_rate from arguments and using the one already assigned to the builder

Reply:
done